docs/documentation/using-notify.md

```diff
@@ -0,0 +1,48 @@
+To set it locally, run this command in your Terminal, in the root
+folder of your prototype:
+```shell
+   echo NOTIFYAPIKEY=xxxxxxx >> .env
+````   
+(where xxxxxxx is a key you’ve created in Notify). This creates a file 
```

Comment:
In most cases, .env should already exist thanks to https://github.com/alphagov/govuk-prototype-kit/blob/9953d9d5b5b5456e1ce5e76d731ac31c55823d48/start.js#L46-L52

If I had already customised that file and didn't know what that command did, I'd be nervous that it'd wipe out my existing env configuration.